src/pages/node-express/lesson8-a1.mdx

```diff
@@ -0,0 +1,53 @@
+
+That controller should return a 200 response code and a JSON object that has one attribute, "message", with some message to the user that includes the user's name (from `req.user.name`).
+
+Test your application with Postman. What you test for the `POST` request should be straightforward. But if you try the GET request, you should get back "unauthorized". To test a successful GET request, you have to configure the authorization for that request in Postman. First do the POST request. You'll get back a token. Copy the token to the clipboard. Then go to the GET request and click on Authorization. Select Bearer and paste in the token. Save the request. Then send it. You should get back the message with your user's name.
+
+Optionally, you could create a simple HTML and JavaScript front end for the application in a preferred/public directory. The front end would have a login form, and would make a fetch or axios request to the back end API to do the login, saving the resulting token in local storage. Then it would send another fetch or axios request to the back end, with a Bearer token in the authorization header, to get the hello message, displaying the result to the user. This is only a few lines of work, and you can look at the video instructor's example.
+
+## Alternate Coding Assignment
+
+This is only in case you aren't confident that you can complete the preferred coding assignment. You continue to work in the `node-express-course` repository. For this lesson, you work in the `05-JWT-Basics/starter` directory. Create a new branch, `week8`. **This branch should be created when the `week7` branch is active**. Then run `npm install` to install the required node packages. Duplicate the steps that the instructor performs, testing with Postman as you go. There are two routes to test, a `POST` route to `/api/v1/login`, and a `GET` request, to `/api/v1/dashboard`. For the `GET` request, you have to configure the authorization for that request in Postman. First do the `POST` request and copy the token to the clipboard. Then go to the `GET` request and click on `Authorization`. Select `Bearer` and paste in the token value. Then save the request and try it.
+
+## Saving the Token Automatically in Postman
+
+This is a little complicated, but you should learn to do it for use in future assignments. First click on New in the upper left, and select Environment. Set the title to JWT Basics. Then on the upper right, click on Environments and set it to JWT Basics, to make that environment active.
+
+![Walkthrough of Postman environment intialization](/lesson8-postman-init.gif)
+_(right-click and open image in a new tab to see it a little larger)_
+
+Now open up the POST request for logon. Click on tests. Then enter the following code:
+
+```js
+const jsonData = pm.response.json();
+pm.environment.set("token", jsonData.token);
+```
+
+![Screenshot of setting a variable from the Postman environment](/lesson8-postman-env.png)
+
```

Comment:
Added image and gif walkthrough to help elucidate how to follow these Postman instructions